cukes-ldap/src/main/java/lv/ctco/cukes/ldap/internal/EntityService.java

```diff
@@ -61,6 +62,21 @@ public void deleteEntityByDn(String dn) {
         }
     }
 
+    public List<Attributes> searchByFilter(String dn, String filter){
+        try {
+            LdapContext context = connectionService.getContext();
```

Comment:
The context should be closed in 'finally' block. Refer to other methods implementing the same pattern.

Reply:
Fixed that now.